include/envoy/server/instance.h

```diff
@@ -219,6 +219,13 @@ class Instance {
    */
   virtual std::chrono::milliseconds statsFlushInterval() const PURE;
 
+  /**
+   * Flush the stats sinks outside of a flushing interval.
+   * Note: stats flushing is not synchronous.
+   * Therefore, this function will return prior to flushing taking place.
```

Comment:
s/will/may/

Reply:
important. I also added to the prior sentence. As flushing may be synchronous in some cases. Thanks for paying attention to the details :)